packages/nodes-base/nodes/Form/utils.ts

```diff
@@ -140,8 +140,15 @@ const checkResponseModeConfiguration = (context: IWebhookFunctions) => {
 	}
 };
 
+const authProperty = (nodeType: string) => {
+	if (nodeType === 'n8n-nodes-base.wait') {
+		return 'incomingAuthentication';
```

Comment:
since this is defined in the node already, could we somehow pick it from there instead ?